src/patterns/asking-for-feedback/index.md.njk

```diff
@@ -0,0 +1,142 @@
+
+This guidance applies to all services and products, including those for:
+
+- the public on GOV.UK
+- professional users
+- staff use only
+
+Always follow this advice even if the way that you ask for feedback changes as your service grows.
+
+This advice is technology agnostic and it applies whether you build your own feedback form, purchase one off-the-shelf, or use a third party tool.
+
+### How it works
+
+When asking for feedback, you must consider:
+
+- accessibility
+- how you will create an inclusive service – for example, how might you accidentally exclude someone from giving feedback?
+- data protection
+- design standards - making your page or form work and look the same way as every other page in your service
+- Welsh language - if you offer a Welsh service
+- security - you do not want someone to use your feedback page as an attack vector against the service, organisation, or Government
+
+You must be able to show these things:
+
+- in a Service Assessment
+- at an internal review
+- to someone else who asks how you're meeting your legal duties
+
+## Before you create a new feedback page or form
+
+For services on GOV.UK, you should check the [creating a standard feedback page on GOV.UK](https://www.gov.uk/service-manual/service-assessments/get-feedback-page). This is part of the process for getting your service onto a GOV domain.
+
+Services and products not on GOV.UK should consider the ‘standard’ feedback page first.
+
+You should only differ from that where you can demonstrate that it does not meet the users need, or you have a better way to meet them.
+
+For example, if you need to ask more specific questions so that users can answer the questions more successfully.
+
+### If you create your own form or page
+
+The following roles should be involved in creating new forms or pages:
+
+1. A user researcher should lead the discussion of ‘what we need to know’ and ‘information to collect’. They must make sure that it is tested with real end users.
+2. A content designer should lead the design of the form.
+3. Use an interaction designer if you need to, to ensure existing proven patterns are used or to create new patterns.
+
+The rest of the development team should follow their same responsibilities they would for any other part of your service. For example, the feedback form must be accessibility tested for compliance.
+
+### How to decide between third party products or building your own
+
+Off-the-shelf surveys and form products, for example Microsoft forms and Google forms, are often used for collecting information.
+
+The laws that apply to feedback pages we code ourselves apply equally to those we choose to include from third parties. As the operator of the service, we are responsible for those third parties compliance with the law. For example the data protection laws and the Equality Act 2010.
```

Comment:
Not sure I understand what this paragraph is trying to say. If we use a third party tool (e.g. Google Analytics) then it isn't our responsibility to ensure their compliance.